opm/simulators/linalg/findOverlapRowsAndColumns.hpp

```diff
@@ -110,6 +110,28 @@ namespace detail
             }
         }
     }
+    template <class Grid>
+    size_t numInteriorCells(const Grid& grid, bool ownerFirst)
+    {
+        size_t numInterior = 0;
+        if (!ownerFirst)
```

Comment:
Add least with the PreconditionerFactory it seems to be possible to construct a ParallelOverlappingILU0 in a serial run. Maybe change to `if (!ownerFirst || grid.com().size()==1)`

Reply:
This is added.